tests/test_views.py

```diff
@@ -1624,6 +1628,7 @@ def setUp(self):
         self.project = ProjectFactory(creator=self.creator)
         # Authenticate all requests
         self.app.authenticate(*self.creator.auth)
+        contributor_added.connect(notify_added_contributor)
```

Comment:
Why is that? I thought the signal needs to be re-connected here because it was disconnected in the `AppTestCase` class